functions-app/school-results-load-cache/function.json

```diff
@@ -0,0 +1,10 @@
+{
+  "bindings": [
+    {
+      "name": "myTimer",
+      "type": "timerTrigger",
+      "direction": "in",
+      "schedule": "0 */1 * * * *"
```

Comment:
can you add a property to describe the schedule please.  think i did this on the school pin gen one in tslib

Reply:
Done